src/System.Threading.ThreadPool/tests/ThreadPoolTests.cs

```diff
@@ -52,8 +55,8 @@ public static void GetMinMaxThreadsTest()
         {
             int minw, minc;
             ThreadPool.GetMinThreads(out minw, out minc);
-            Assert.True(minw > 0);
-            Assert.True(minc > 0);
+            Assert.True(minw >= 0);
+            Assert.True(minc >= 0);
```

Comment:
@kouvel Returning Environment.ProcessorCount would be technically a lie.  We may still do that if we had evidence that would improve compatibility with real-world apps.